Workspace/src/ik/src/Roboarm.cpp

```diff
@@ -16,27 +21,27 @@ Roboarm::Roboarm(double lengths[3], double angles[3], double stepSize)
   }
 }
 
-bool Roboarm::calculateVelocities(double endEffector[3], double angles[3],
+bool Roboarm::calculateVelocities(double endEffectorVel[3], double curAngles[3],
                                   bool limits) {
 
   // prevent elbow from fully extending (0 radian to shoulder link) or
   // determinant will be 0.
-  if (abs(angles[1]) < 0.0872665) { // 5 degree tolerance
+  if (abs(curAngles[1]) < 0.0872665) { // 5 degree tolerance
     cout << "Inverse jacobian unsolved. Halting motion";
     linkVelocities[0] = 0.0;
     linkVelocities[1] = 0.0;
     linkVelocities[2] = 0.0;
     return false;
   }
 
-  double eeX = endEffector[0];
-  double eeY = endEffector[1];
-  double eePhi = endEffector[2];
+  double eeX = endEffectorVel[0];
```

Comment:
Does this stand for end effector x, y, phi?

Reply:
yeah, I kept the naming convention from the original implementation